deployment&configuration/netsuite/flows/shipping-method-mapping/readme.md

```diff
@@ -0,0 +1,38 @@
+# Shipping Method Mapping between HotWax Commerce and NetSuite
+
+The shipping method mapping between HotWax Commerce and NetSuite ensures that the shipping methods selected in Shopify correspond accurately to their counterparts in both HotWax Commerce and NetSuite. HotWax Commerce facilitates the mapping of shipping methods chosen from Shopify, providing accurate information to NetSuite. This process establishes coherence across platforms, maintaining consistency in order processing and fulfillment workflows. 
```

Comment:
Check this document, why exactly do we need a [Shipping method mapping](https://docs.hotwax.co/krewe/flows/salesorders/shippingmethmapping)?